tests/test_backend.py

```diff
@@ -46,96 +45,72 @@ def __call__(self, message, dkim_domain=None, dkim_key=None,
         return self.configuration_set
 
 
-class FakeSESConnection(object):
-    """
+class FakeSESConnection:
+    '''
     A fake SES connection for testing purposes.It behaves similarly
     to django's dummy backend
     (https://docs.djangoproject.com/en/dev/topics/email/#dummy-backend)
 
     Emails sent with send_raw_email is stored in ``outbox`` attribute
     which is a list of kwargs received by ``send_raw_email``.
-    """
+    '''
     outbox = []
 
     def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
-
-        self.connection = boto3.client(
-            'ses',
-            aws_access_key_id='ACCESS_KEY',
-            aws_secret_access_key='SECRET_KEY',
-        )
-
         self.outbox.append(kwargs)
 
     def send_raw_email(self, **kwargs):
         self.outbox.append(kwargs)
-        return {
+        response = {
             'MessageId': 'fake_message_id',
             'ResponseMetadata': {
                 'RequestId': 'fake_request_id',
-            },
-        }
-
-    def send_mail(self, subject, body, from_email, recipient_list):
-        with mock.patch('boto3.client', return_value=self.connection):
-            with mock.patch.object(self.connection, 'send_raw_email', return_value=self.send_raw_email()):
-                # todo not working because we dont call the actual function -> need to override inside send_raw_email
-                send_mail(subject, body, from_email, recipient_list)
-        self.outbox.append(EmailMultiAlternatives(subject, body, from_email, recipient_list))
+                },
+            }
+        return response
 
 
 class FakeSESBackend(django_ses.SESBackend):
-    """
+    '''
     A fake SES backend for testing purposes. It overrides the real SESBackend's
     get_rate_limit method so we can run tests without valid AWS credentials.
-    """
+    '''
 
     def get_rate_limit(self):
         return 10
 
 
-class SESBackendBaseTest(TestCase):
-
-    @classmethod
-    def setUpTestData(cls):
-        super().setUpTestData()
-
-        cls.fake_connection = FakeSESConnection()
-
-        cls.connection = boto3.client(
-            'ses',
-            aws_access_key_id='ACCESS_KEY',
-            aws_secret_access_key='SECRET_KEY',
-        )
+class SESBackendTest(TestCase):
+    def setUp(self):
+        # TODO: Fix this -- this is going to cause side effects
+        django_settings.EMAIL_BACKEND = 'tests.test_backend.FakeSESBackend'
+        django_ses.boto3.client = FakeSESConnection
+        self.outbox = FakeSESConnection.outbox
 
     def tearDown(self):
-        # Empty outbox every time test finishes
-        self.fake_connection.outbox = []
+        # Empty outbox everytime test finishes
```

Comment:
you replaced my typo fix :)

Reply:
My mistake, I started from a fresh file and forgot some of your changes apparently.